src/main/java/com/vocabsvc/wordtrainer/dto/WordDto.java

```diff
@@ -0,0 +1,25 @@
+package com.vocabsvc.wordtrainer.dto;
+
+import com.vocabsvc.wordtrainer.model.Word;
+import java.util.ArrayList;
+import javax.validation.constraints.NotNull;
+import lombok.Getter;
+import lombok.Setter;
+
+@Setter
+@Getter
+public class WordDto {
```

Comment:
I think it would be better for DTOs to have more descriptive names such as WordRequest/WordResponse.
But as it seems we are using the same DTO for both request and response, a suffix of DTO might suffice for now.
It is generally preferred to leave acronyms in all caps, so I'd suggest WordDTO over WordDto.

Reply:
yeah, agreed .i think It would also make more sense if i refactored the `userDTO` to `userPayload`, looks more meaningful to me.